src/DssSpell.t.sol

```diff
@@ -121,10 +121,10 @@ contract DssSpellTest is DSTest, DSMath {
 
```

Comment:
Can't tag the exact piece of code, but you can remove the `LPTokenLike` interface at the top of this file.

Reply:
done